src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 101;
+    private Random value = new Random();
```

Comment:
`Random` is not a value yet, it is a class with a functionality that can provide us with values
```suggestion
    private Random random = new Random();
```